components-api/src/main/scala/pl/touk/nussknacker/engine/api/typed/CanBeSubclassDeterminer.scala

```diff
@@ -24,6 +24,8 @@ trait CanBeSubclassDeterminer {
     (givenType, superclassCandidate) match {
       case (_, Unknown) => ().validNel
       case (Unknown, _) => ().validNel
+      case (TypedNull, _) => ().validNel
+      case (_, TypedNull) => s"No type can be subclass of ${TypedNull.display}".invalidNel
```

Comment:
What about  union with Null?

Reply:
Union must be made out of types that we know something about and we do not know anything about type of Null. If I allowed Null in union that there would be a problem with things like method suggestions on FE.